great_expectations/datasource/data_connector/configured_asset_azure_data_connector.py

```diff
@@ -0,0 +1,17 @@
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetFilePathDataConnector,
+)
+
+
+class ConfiguredAssetAzureDataConnector(ConfiguredAssetFilePathDataConnector):
+    def __init__(self):
+        pass
+
+    def build_batch_spec(self):
+        pass
+
+    def _get_data_reference_list_for_asset(self):
+        pass
+
+    def _get_full_file_path(self):
+        pass
```

Comment:
Stub class just to get everything up and running (actual details to come in a following PR!)